src/index.js

```diff
@@ -40,7 +40,7 @@ const init = async () => {
     },
   }));
 
-  let assetsUrl = config.hostingEnvironment.assetsUrl || 'https://rawgit.com/DFE-Digital/dfe.ui.toolkit/master/dist/';
+  let assetsUrl = config.assets.url;
```

Comment:
is this now changed in our config repos?

Reply:
Changed for other repos, I was going to do it for this one now